framework/src/org/apache/cordova/PluginManager.java

```diff
@@ -381,15 +374,16 @@ public void onReset() {
     }
 
     Uri remapUri(Uri uri) {
+        Uri remap = uri;
         for (CordovaPlugin plugin : this.pluginMap.values()) {
             if (plugin != null) {
-                Uri ret = plugin.remapUri(uri);
+                Uri ret = plugin.remapUri(remap);
                 if (ret != null) {
-                    return ret;
+                    remap = ret;
                 }
             }
         }
-        return null;
+        return uri.equals(remap) ? null : remap;
```

Comment:
It was the default behavior to return null if no plugins ends up remapping the uri. The only change is that it also return null if a plugin remap the uri to the same uri, because there's no need to act like there's a change when there's not (some plugins like JsHybuggerPlugin for example always remap every uri, to the same when no action is needed)